content/pt/docs/concepts/signals/metrics.md

```diff
@@ -0,0 +1,128 @@
+
+- **Counter**: Um valor que acumula com o tempo -- você pode imaginar isso como
+  um odômetro de um carro; é um valor que só cresce.
+- **Asynchronous Counter**: Assim como o **Counter**, porém é coletado uma vez a
+  cada exportação. Pode ser usado em casos onde você não tenha acesso aos
+  incrementos contínuos, mas apenas ao valor agregado.
+- **UpDownCounter**: Um valor que acumula com o tempo, mas também pode cair. Um
+  exemplo seria o tamanho de uma fila, este valor irá aumentar e diminuir de
+  acordo com o número de itens que estão entrando ou saindo desta fila.
+- **Asynchronous UpDownCounter**: Assim como o **UpDownCounter**, porém é
+  coletado uma vez a cada exportação. Pode ser usado em casos onde você não
+  tenha acesso às mudanças contínuas, mas apenas ao valor agregado (ex., atual
+  tamanho da fila).
+- **Gauge**: Mede o valor atual no momento da leitura. Um exemplo seria um
+  medidor de tanque de combustível de um veículo. Gauges são assíncronos.
+- **Histogram**: Uma agregação de valores client-side, ta como latências de
+  requisições. Um histograma é uma boa escolha se você está interessado em
+  valores de estatísticas. Por exemplo: Quantas requisições estão levando menos
+  de 1s?
+
+Para visualizar mais instrumentos síncronos, assíncronos, e entender qual dos
+tipos melhor se encaixa no seu caso de uso, veja
+[Diretrizes Suplementares](/docs/specs/otel/metrics/supplementary-guidelines/).
+
+## Agregação {#aggregation}
+
+Em adicional aos instrumentos de métrica, também é importante entendermos o
+conceito de **agregações**. Uma agergação é uma teçnica pela qual um grande
+número de medições é combinado em estatísticas exatas ou estimadas sobre eventos
+de métricas que ocorreram durante uma janela de tempo. O protocolo OTLP
+transporta essas métricas agregadas. A API do OpenTelemetry fornece uma
+agregação padrão para cada instrumento, que podem ser sobrescritas com o uso de
+Views. Por padrão, o projeto OpenTelemetry visa fornecer agregações que sejam
+suportadas por diferentes visualizadores e backends de telemetria.
+
+Ao contrário dos [rastros](/docs/concepts/signals/traces/), que são destinados a
+capturar os ciclos de vida das requisições e fornecer o contexto para as partes
+individuais de uma requisição, as métricas são destinadas a fornecer informações
+estatísticas em forma de dados agregados. Alguns exemplos de caso de uso para as
+métricas incluem:
+
+- Reportar o número total de bytes lidos por um serviço, por tipo de protocolo.
+- Reportar o número total de bytes lidos e os bytes por requisição.
+- Reportar a duração de uma chamada de sistema.
+- Reportar tamanhos de requisições para determinar uma tendência.
+- Reportar o uso de CPU ou memória de um processo.
+- Reportar valores médios de saldo de uma conta.
+- Reportar o número atual de requisições ativas sendo processadas.
+
+## Views
+
+Uma view oferece aos usuários a flexibilidade de personalizar a saída das
```

Comment:
```suggestion
Uma _view_ oferece aos usuários a flexibilidade de personalizar a saída das
```
Os termos em inglês que não estamos traduzindo estamos botando em itálico. Se puder refeletir pra todos no texto seria massa!

Reply:
Isso também incluiria os nomes de componentes, por exemplo, Collector e Meter Provider?